bindings/python/crocoddyl/core/integrator/euler.cpp

```diff
@@ -77,10 +77,14 @@ void exposeIntegratedActionEuler() {
           "differential",
           bp::make_getter(&IntegratedActionDataEuler::differential, bp::return_value_policy<bp::return_by_value>()),
           "differential action data")
+      .add_property(
+          "control",
+          bp::make_getter(&IntegratedActionDataEuler::control, bp::return_value_policy<bp::return_by_value>()),
+          "control parametrization data")
       .add_property("dx", bp::make_getter(&IntegratedActionDataEuler::dx, bp::return_internal_reference<>()),
                     "state rate.")
       .add_property("Lwu", bp::make_getter(&IntegratedActionDataEuler::Lwu, bp::return_internal_reference<>()),
-                    "Hessian of the cost wrt the differential control (w) and the control parameters (u).");
+                    "hessian of the cost wrt the differential control (w) and the control parameters (u).");
```

Comment:
Capital H was correct (cf. [here](https://en.wikipedia.org/wiki/Hessian_matrix))

Reply:
Yes, nicely catch.

Done (I will push it soon)